### DISCUSSION TITLE
stoicism/knobs/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#26] [General]
# stoicism/knobs/

https://shanenull.com/stoicism/knobs/

**Comment by shane0** [comment]
# ai

- open source ai for science is the most vital requirement for survival of our species

## low hanging fruit

- knobs demonstrating ways ai polarizes people by nature
- just like deep mind played the long game sacrificing early and mid game ai will do the same to manipulate people into purchases and general brainwashing to create compliant sheep
- proprietary ai will be used for evil before good
- military
- not just ai fighting drones but to instigate war by manipulating information
- ai auto complete is the first subtle entry to thought control
- suggesting sentence completions or canned buttons to insert prompts
- prompts are analyzed using sentiment analysis and check emotions with sentiment analysis
- subtle censorship: all current ai try to misdirect the conversation if you attempt to find information that could impact ad revenue
- sentiment biases responses and certain words in prompts generate subtle conversation manipulation
- i have many screen recordings with examples but I feel like I’m helping them become more subtle at brainwashing people so I may stop demonstrating it

## planck blackbody knob page

- https://youtu.be/oEoe6YdUeAE

![blackbody](https://github.com/user-attachments/assets/38575a4b-a576-47bf-ac2b-08d841d3043d)


## poly crisis

- robot knob
- self driving vehicles knob
- healthcare knobs
- education knobs
- homelessness / housing knobs
- nuke knobs https://youtu.be/ujfC0NgdU48
- ecological overshoot knobs
- war knobs
- economic wars
- tech wars
- energy wars

**Comment by shane0** [comment]
# evolving beyond maladapted brains

- https://youtu.be/YBkl2kCdW-w
